CMakeLists.txt

```diff
@@ -115,6 +117,25 @@ else()
     include_directories(${ZSTD_INCLUDE_DIR})
     list(APPEND THIRDPARTY_LIBS ${ZSTD_LIBRARIES})
   endif()
+
+  if(WITH_KAFKA)
+    # System lib used
+    # rdkafka uses pkgconfig instead of Findrdkafka.cmake
+    # https://stackoverflow.com/questions/29191855/what-is-the-proper-way-to-use-pkg-config-from-cmake
+    # find_package(rdkafka REQUIRED)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(RDKAFKA REQUIRED rdkafka++)
+    add_definitions(-DUSE_KAFKA)
+    include_directories(${RDKAFKA_INCLUDE_DIR})
+    list(APPEND THIRDPARTY_LIBS ${RDKAFKA_LIBRARIES})
+  endif()
+
+  if(WITH_AWS)
+    add_definitions(-DUSE_AWS)
+    list(APPEND THIRDPARTY_LIBS aws-cpp-sdk-s3 aws-cpp-sdk-kinesis aws-cpp-sdk-core)
+    include_directories(../../aws-sdk-cpp/aws-cpp-sdk-s3/include ../../aws-sdk-cpp/aws-cpp-sdk-kinesis/include ../../aws-sdk-cpp/aws-cpp-sdk-core/include)
+    link_directories(../../aws-sdk-cpp/aws-cpp-sdk-s3 ../../aws-sdk-cpp/aws-cpp-sdk-kinesis ../../aws-sdk-cpp/aws-cpp-sdk-core)
+  endif()
```

Comment:
I think we will be able to offer a better way to work with AWS SDK, soon.

Reply:
@spetrunia did you made any progress with that ?